### PR TITLE
Challenge

### PR DESCRIPTION
**Pull request by tom-fitz**
@morecallan

### CODE REVIEW

File: tree.js [hunk 1]
```diff
@@ -25,13 +24,12 @@ function treeMaker(tree){
   }
 }
 
-document.addEventListener("keypress", enterKeyPress);
-
 function enterKeyPress(keypress) {
 	if (keypress.which === 13) {
-    console.log("ioeroj");
-		treeMaker();
+    console.log("testing to see if keypress is working.");
```
Comment (morecallan) [inline tree.js]: take this console.log out (your wouldn't have a test console.log like this in "production" like it is on master.

**Review by morecallan — COMMENTED** [review]
@basictom - only one small change!

**Comment by tom-fitz** [comment]
@morecallan Fine. Have it your way.

**Comment by morecallan** [comment]
👍 all good!

**Pull request merged**